src/jit/codegencommon.cpp

```diff
@@ -4334,50 +4361,35 @@ void            CodeGen::genFnPrologCalleeRegArgs(regNumber xtraReg,
                      (varDsc->lvType == TYP_LONG && varDsc->lvOtherReg == REG_STK && regArgTab[argNum].slot == 2));
 
         var_types storeType = TYP_UNDEF;
+        unsigned  slotSize = TARGET_POINTER_SIZE;  // with a HFA this can be changed
```

Comment:
The comment seems unnecessary since storType also will be changed for HFA. They are just default values for non-HFA case.